docs/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+#### Expired ticket
+
+Tickets that reached the end of their window without being called to vote - these can be revoked, but do not grant a reward.
+
+#### Faucet
+
+A mechanism for obtaining free (testnet) tokens.
+
+#### Hash
+
+The output of a cryptographic hashing function that produces a fixed-size and unique hash value from variable-size input. 
+
+#### Hash function
+
+A cryptographic function that produces a fixed-size and unique hash value from variable-size transaction input. Decred uses the [BLAKE-256](https://docs.decred.org/research/blake-256-hash-function/) hashing function. 
+
+#### Hashrate
+
+The number of hashes per second computed by miners on the network.
+
+#### Hybrid PoW/PoS
+
+In Decred, a [hybrid PoW/PoS system](https://docs.decred.org/research/hybrid-design/) is used, whereby blocks mined by PoW miners must be approved by ticket holders, which are pseudorandomly selected from the ticket pool. This provides a check on PoW miners and increases the overall cost of attacking the network. When a block is approved, 60% of the block reward goes to the PoW miner, 30% goes to the holders of  tickets called to vote (approve blocks), and 10% goes to the Decred Treasury to fund the project's development.
+
+#### Immature ticket
+
+Once tickets are mined they are immature for 256 blocks (about 20 hours) and cannot be called to vote until after this period elapses.
+
+#### Inflation
+
+Increase in the [available supply](https://docs.decred.org/advanced/inflation/) of Decred as new DCR is minted into existence through the block reward.
+
+#### Live ticket
+
+Tickets that are waiting to be called.
+
+#### Mainnet
+
+The proper Decred network - a term used to differentiate from testnet.
+
+#### Mempool
+
+Transactions waiting to be mined.
+
+#### Miners
+
+Miners help secure the network via Proof-of-Work (PoW) mining. Miners recieve block rewards and transaction fees for successfully validating blocks. 
+
+#### Mining Pool
+
+A group of miners who share (pool) their computational resources to mine DCR. When a member of a mining pool successfully mines a block, the rewards are split among all pool members in proportion to the hashpower they contribute to the pool. 
+
+#### Missed ticket
+
+Tickets that have been called but did not respond - these can be revoked, but do not grant a reward.
+
+#### Orphan
+
+A valid block which is not built upon and therefore not included in the definitive blockchain.
+
+#### Politeia
+
+A system for facilitating the submission and discussion of [proposals](https://proposals.decred.org/) in an environment with transparent censorship. 
+
+#### Private key
+
+A string of data that allows you to access and spend DCR stored in address associated with the private key. When sending DCR from one address to another, the wallet must use the private key of the sending address to sign the transaction.
```

Comment:
Don't call this a string.  It's just binary data (actually, it's a number, but it's encoded as binary).

DCR can not be sent "from" an address.  Addresses are only capable of receiving coins, not sending.

Reply:
Also, the DCR is not "stored" in an address.  Perhaps `An astronomically large secret number that allows DCR to be spent.  All transactions that spend coins must include a signature that can only produced with knowledge of the secret number."

Reply:
Going with @davecgh 's definition:

#### Private key

An astronomically large secret number that allows DCR to be spent. All transactions that spend DRC must include a signature that can only be produced with knowledge of this secret number.